internal/install/execution/recipe_var_provider.go

```diff
@@ -77,18 +76,14 @@ func (re *RecipeVarProvider) Prepare(m types.DiscoveryManifest, r types.OpenInst
 	return vars, nil
 }
 
-func varsFromProfile(licenseKey string) (types.RecipeVars, error) {
+func varsFromProfile() (types.RecipeVars, error) {
 	accountID := configAPI.GetActiveProfileString(config.AccountID)
 	apiKey := configAPI.GetActiveProfileString(config.APIKey)
 	region := configAPI.GetActiveProfileString(config.Region)
 
-	if licenseKey == "" {
-		return types.RecipeVars{}, errors.New("license key not found")
-	}
-
 	vars := make(types.RecipeVars)
 
-	vars["NEW_RELIC_LICENSE_KEY"] = licenseKey
+	vars["NEW_RELIC_LICENSE_KEY"] = os.Getenv("NEW_RELIC_LICENSE_KEY")
```

Comment:
Since we're not passing the licenseKey as a parameter any more, wondering if we should follow suit and use `configApi.GetActiveProfileString(config.LicenseKey)`, similar to the other attributes from profile